src/cljdoc/render/api.clj

```diff
@@ -34,11 +34,13 @@
 (defn docstring->html [doc-str render-wiki-link]
   [:div
    [:div.lh-copy.markdown
-    (-> doc-str
-        (rich-text/markdown-to-html
-         {:escape-html? true
-          :render-wiki-link (comp render-wiki-link parse-wiki-link)})
-        hiccup/raw)]
+    (if (= doc-str false)
+      ""
+      (-> doc-str
```

Comment:
You could also just do
```clj
(when doc-str
  (-> doc-str ,,,))
```
In hiccup an empty string and nil are treated identically in most cases I think.

Reply:
Wasn't sure if `nil` works as result here. Changed now and result still looks the same :)